changelog/action.yml

```diff
@@ -43,7 +51,9 @@ runs:
           --wp-endpoint='${{ inputs.endpoint }}' \
           --wp-status='${{ inputs.status }}' \
           --wp-tag-ids='${{ inputs.tag-id }}' \
-          --link-to-pr='${{ inputs.link-to-pr }}'
+          --link-to-pr='${{ inputs.link-to-pr }}' \
+          --start-marker='${{ inputs.start-marker }}' \
+          --end-marker='${{ inputs.end-marker }}'
```

Comment:
My only concern is that bad things may happen if any of the markers contain an apostrophe; it will be possible to inject a (malicious) command into the job.

Reply:
interesting.. but that's true for any string parameters, right?

Reply:
Yes.

Please check the Remediation section of https://securitylab.github.com/research/github-actions-untrusted-input/